projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+    "subtitle.refurbished_furniture.microwave.open": "微波炉：打开",
+    "subtitle.refurbished_furniture.microwave.close": "微波炉：关闭",
+    "subtitle.refurbished_furniture.kitchen_drawer.open": "抽屉：滑动打开",
+    "subtitle.refurbished_furniture.kitchen_drawer.close": "抽屉：滑动关闭",
+    "subtitle.refurbished_furniture.cutting_board.place_ingredient": "物品：放置",
+    "subtitle.refurbished_furniture.kitchen_sink.fill": "水：流动",
+    "subtitle.refurbished_furniture.fridge.open": "冰箱：打开",
+    "subtitle.refurbished_furniture.fridge.close": "冰箱：关闭",
+    "subtitle.refurbished_furniture.stove.open": "炉灶：打开",
+    "subtitle.refurbished_furniture.stove.close": "炉灶：关闭",
+    "subtitle.refurbished_furniture.freezer.open": "冰柜：打开",
+    "subtitle.refurbished_furniture.freezer.close": "冰柜：关闭",
+    "subtitle.refurbished_furniture.drawer.open": "木质抽屉：打开",
+    "subtitle.refurbished_furniture.drawer.close": "木质抽屉：关闭",
+    "subtitle.refurbished_furniture.workbench.craft": "物品：建造",
+    "subtitle.refurbished_furniture.wrench_selected_node": "节点：高亮",
+    "subtitle.refurbished_furniture.wrench_remove_link": "链路：移除",
+    "subtitle.refurbished_furniture.wrench_connected_link": "链路：连接",
+    "subtitle.refurbished_furniture.knife_chop": "刀：切碎",
+    "subtitle.refurbished_furniture.spatula_scoop": "锅铲：舀勺",
+    "subtitle.refurbished_furniture.ui.paddle_ball.retro_click": "怀旧游戏：点击",
+    "subtitle.refurbished_furniture.ui.paddle_ball.retro_hit": "怀旧游戏：击中",
+    "subtitle.refurbished_furniture.ui.paddle_ball.retro_success": "怀旧游戏：成功",
+    "subtitle.refurbished_furniture.ui.paddle_ball.retro_fail": "怀旧游戏：失误",
+    "subtitle.refurbished_furniture.ui.paddle_ball.retro_win": "怀旧游戏：胜利",
+    "subtitle.refurbished_furniture.ui.paddle_ball.retro_lose": "怀旧游戏：失败",
+    "computer_program.refurbished_furniture.paddle_ball": "乒乓",
+    "computer_program.refurbished_furniture.paddle_ball.play_ai": "对战AI",
+    "computer_program.refurbished_furniture.paddle_ball.play_vs": "对战玩家",
+    "computer_program.refurbished_furniture.paddle_ball.main_menu": "主菜单",
+    "computer_program.refurbished_furniture.paddle_ball.you": "你",
+    "computer_program.refurbished_furniture.paddle_ball.win_game": "你赢了 :)",
+    "computer_program.refurbished_furniture.paddle_ball.lose_game": "你输了 :(",
+    "computer_program.refurbished_furniture.paddle_ball.searching_players": "寻找对手中……",
+    "computer_program.refurbished_furniture.paddle_ball.opponent_left": "对手离开了游戏",
+    "computer_program.refurbished_furniture.paddle_ball.cancel": "取消",
+    "computer_program.refurbished_furniture.paddle_ball.server_required": "必须在服务器上游玩",
+    "computer_program.refurbished_furniture.home_control": "智能家居",
+    "computer_program.refurbished_furniture.home_control.turn_on_all": "全部开启",
+    "computer_program.refurbished_furniture.home_control.turn_off_all": "全部关闭",
+    "computer_program.refurbished_furniture.home_control.info": "家居控制允许你控制连接在电力网络上的设备的电源，设备必须连接到与电脑相同的网络才能被发现，在放置前使用铁砧重命名设备，以便在应用中更容易识别。",

```

```suggestion
    "computer_program.refurbished_furniture.home_control.info": "“家居控制”用于控制接电设备的电源。设备需与电脑处于同一网络才可控制。在放置设备前可使用铁砧重命名，以便在应用中识别。",
```
建议（可不更改）
算是整体乱改了一通

如不更改，也建议`家居控制`两侧加上中文双引号